CK3toEU4/Source/Mappers/ProvinceMapper/ProvinceMapper.h

```diff
@@ -0,0 +1,41 @@
+#ifndef PROVINCE_MAPPER_H
+#define PROVINCE_MAPPER_H
+
+#include "ProvinceMappingsVersion.h"
+#include "Parser.h"
+#include <map>
+
+
+namespace CK3
+{
+class Title;
+}
+
+namespace mappers
+{
+class ProvinceMapper: commonItems::parser
```

Comment:
Should expand the tests to cover the newer functions like looking up using county name or looking up titles.